public/app/features/dashboard-scene/edit-pane/DashboardEditPane.tsx

```diff
@@ -109,11 +118,6 @@ export class DashboardEditPane extends SceneObjectBase<DashboardEditPaneState> {
 
   public clearSelection() {
     const dashboard = getDashboardSceneFor(this);
-
```

Comment:
Are we removing the fix on purpose?

Reply:
@bfmatei no, must have been switching branches, and happened by mistake